File: src/app/models/dtos/air-index.dto.ts
import {AirIndexLevelDto} from './air-index-level.dto';

export interface AirIndexDto {
  id: number;
  summary: AirIndexLevelDto;
  benzene: AirIndexLevelDto;
  so2: AirIndexLevelDto;
  co: AirIndexLevelDto;
  o3: AirIndexLevelDto;
  pm10: AirIndexLevelDto;
  pm25: AirIndexLevelDto;
  no2: AirIndexLevelDto;
}

File: backend/Dto/AirIndexDto.cs
namespace AirTrackerAPI.Dto
{

  public class AirIndexDto
  {
    public int Id { get; set; }
    public AirIndexLevel Summary { get; set; }
    // C6H6
    public AirIndexLevel Benzene { get; set; }
    public AirIndexLevel SO2 { get; set; }
    public AirIndexLevel CO { get; set; }
    public AirIndexLevel O3 { get; set; }
    public AirIndexLevel PM10 { get; set; }
    public AirIndexLevel PM25 { get; set; }
    public AirIndexLevel NO2 { get; set; }
  }
}
